docs/DeveloperGuide.md

```diff
@@ -369,16 +371,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2.  Should be able to hold up to 1000 internship entries without a noticeable sluggishness in performance for typical usage.
+3.  An undergraduate student with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 
 *{More to be added}*
 
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
-* **Private contact detail**: A contact detail that is not meant to be shared with others
-
+* **Computing undergraduate**: Refers to an undergraduate of the National University of Singapore who is currently studying towards a computing degree.
```

Comment:
Why specifically NUS? Would it suffice to mention _Refers to a university undergraduate pursuing a computing degree_?

Reply:
I agree.

Reply:
Agree